lighthouse/classes/events/biosero/source_completed.py

```diff
@@ -52,3 +53,10 @@ def _create_message(self) -> Any:
             self.properties[property_name].add_to_warehouse_message(message)
 
         return message.render()
+
+    def process_event(self) -> None:
+        super().process_event()
+
+        response = self.transfer_to_bin()
+        if not response.ok:
```

Comment:
Shouldn't this catch and message be in the `transfer_to_bin()` method instead?

Reply:
Yes, makes sense, as all services should raise they're exceptions

Reply:
Done